tests/examples/test1/dependencies.yaml

```diff
@@ -0,0 +1,69 @@
+files:
+  all:
+    generate: [conda, requirements]
+    requirements_dir: output/actual
```

Comment:
Perhaps document how to update these tests somewhere: run the generator, manually verify the results, and copy `output/actual` to `output/expected`.

Reply:
Whoops. Forgot to do this in my latest commits. I'll tackle this next.

Reply:
updated in d61da2e